components/scream/src/physics/rrtmgp/atmosphere_radiation.cpp

```diff
@@ -88,7 +89,7 @@ void RRTMGPRadiation::set_grids(const std::shared_ptr<const GridsManager> grids_
     if (it == "h2o") { /* Special case where water vapor is called h2o in radiation */
       // do nothing, qv has already been added.
     } else {
-      add_field<Required>(it,scalar3d_layout_mid,kgkg,grid_name, ps);
+      add_field<Computed>(it,scalar3d_layout_mid,kgkg,grid_name, ps);
```

Comment:
I think we still need o2 and o3 to be "required", since those will still be read from file. Once we can remap online, I think they can also be computed and read by this process.